### PR TITLE
README: update for Go 1.21 release

### PR DESCRIPTION
**Pull request by dgryski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,28 @@
 # compute-sdk-go
 
-Experimental Go SDK for building [Compute@Edge](https://www.fastly.com/products/edge-compute/serverless) applications with [TinyGo](https://tinygo.org/).
+Experimental Go SDK for building [Compute@Edge](https://www.fastly.com/products/edge-compute/serverless) applications with [Go](https://go.dev) (1.21+) and [TinyGo](https://tinygo.org/) (0.28.1+).
```
Comment (joeshaw) [inline README.md:3]: Can we drop "experimental" yet?

**Review by joeshaw — APPROVED** [review]
LGTM, one question below.

File: README.md (hunk 1 above)
Comment (dgryski) [reply README.md:3]: Almost certainly.  (Unless we want to leave it until we tag 1.0)

**Comment by dgryski** [comment]
We should probably mention `[scripts.build]`

**Review by joeshaw — APPROVED** [review]
LGTM.

I hope that the `fastly` CLI and/or `fastly.toml` will make it easier to switch between TinyGo and Go without needing to mess with `scripts.build` but in the meantime I think this is a good thing to mention.

**Pull request merged**